src/test/TestUtils.js

```diff
@@ -116,25 +110,43 @@ async function printReleaseDiff(res, query) {
             url: `${process.env.RELEASE_URL}${query}`,
             qsStringifyOptions: { arrayFormat: 'repeat' },
         };
-        const rel = await RequestUtils.createRequest(options);
-        if (JSON.stringify(res) !== rel) {
-            let relJSON;
+        const releaseStr = await RequestUtils.createRequest(options);
+        const responseStr = JSON.stringify(res);
+        if (responseStr !== releaseStr) {
+            let releaseJSON;
             try {
-                relJSON = JSON.parse(rel);
-                const resCpy = JSON.parse(JSON.stringify(res.data[0] || res));
-                removeKeys(relJSON, ['path', 'stops']);
-                removeKeys(resCpy, ['path', 'stops']);
-                const delta = JsonDiff.diff(relJSON.data[0] || relJSON, resCpy);
+                releaseJSON = JSON.parse(releaseStr);
+                const responseJSONCopy = JSON.parse(responseStr);
+                const delta = JsonDiff.diff(releaseJSON, responseJSONCopy);
                 jsondiffpatch.console.log(delta);
             } catch (e) {
-                console.log(`WARNING: Current release: ${rel}\nLocal: ${JSON.stringify(res)}`);
+                console.log(`WARNING: Current release: ${releaseStr}\nLocal: ${JSON.stringify(res)}`);
```

Comment:
You can use `responseStr` instead of `JSON.stringify(res)`

Reply:
True. Nice catch